module/roi/abstract_roi.hpp

```diff
@@ -2,6 +2,67 @@
 
 #include <fmt/core.h>
 
+#include <iostream>
+#include <opencv2/opencv.hpp>
+
 namespace abstract_roi {
+class Abstract_ImageRoi {
+ protected:
+  cv::Mat roi_img_;
+  cv::Point2d tl_;
```

Comment:
A little aligment would be better.

```suggestion
  cv::Mat     roi_img_;
  cv::Point2d tl_;
```